.ci/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:bionic
+
+RUN apt update && apt install -y --no-install-recommends \
+        build-essential \
+        g++ \
+        cmake \
+        libprotobuf-dev \
+        protobuf-compiler \
+        qt5-default \
+        qttools5-dev \
+        qttools5-dev-tools \
+        qtmultimedia5-dev \
+        libqt5multimedia5-plugins \
```

Comment:
Can you alphabetize the lines? It helps keep it manageable as it changes over time.